test/unit/object/conftest.py

```diff
@@ -116,6 +119,21 @@ def test_collaboration_whitelist_exemption(mock_box_session, mock_object_id):
     return CollaborationWhitelistExemptTarget(mock_box_session, mock_object_id)
```

Comment:
Should we remove: 
- test_collaboration_whitelist 
- test_collaboration_whitelist_entry
- test_collaboration_whitelist_exemption

since the related classes are now deprecated and no longer going to be updated? In other words, these tests could fail at some point in the future, but we wouldn't want to update the to fix it, right?

After you remove these three, just do a quick check to make sure they weren't being called from elsewhere in the  code.

Reply:
Sure, same as `test_collaboration_whitelist_initializer `

Reply:
Decision: keep tests for deprecated methods until they break, then investigate removing both the tests and potentially EOL'ing the deprecated methods, too.